pkg/release/condition_set.go

```diff
@@ -78,7 +78,7 @@ func (c *conditionSet) CRDNotFound(ctx context.Context, crd *apiextensionsv1beta
 				return backoff.Permanent(microerror.Mask(err))
 			}
 
-			return microerror.Maskf(waitError, "CRD %#q in namespace %#q still exists", crd.Name, crd.Namespace)
+			return microerror.Maskf(waitError, "CRD %#q still exists", crd.Name, crd.Namespace)
```

Comment:
How does this look like now?

Reply:
CRD is cluster scope. I saw it somewhere but I don't remember.

Reply:
Then remove the `crd.Namespace` from here :D

Reply:
Ah ok :)
```suggestion
			return microerror.Maskf(waitError, "CRD %#q still exists", crd.Name)
```